prow/cmd/sinker/main.go

```diff
@@ -53,8 +53,11 @@ const (
 	// TODO(fejta): require setting this explicitly
 	defaultConfigPath = "/etc/config/config.yaml"
 
-	reasonAged     = "aged"
-	reasonOrphaned = "orphaned"
+	reasonPodAged     = "aged"
+	reasonPodOrphaned = "orphaned"
+
+	reasonProwJobOldOnce     = "old-once"
```

Comment:
I think we could shorten this to just `"old"`

Reply:
Or, use `"aged"` to mirror what we do for Pods

Reply:
I will do the revision tonight according to your comments.
Please do NOT approve until then.
Thanks.

Reply:
OhOh ... that was too late. :)

Reply:
Sorry, didn't realize we had changed GitHub approval to do LGTM and approve already